test/isochrone.cc

```diff
@@ -69,16 +69,26 @@ void try_isochrone(GraphReader& reader,
                    loki_worker_t& loki_worker,
                    thor_worker_t& thor_worker,
                    const char* test_request,
-                   const std::string& expected) {
+                   const rapidjson::Document& expected) {
   Api request;
   ParseApi(test_request, Options::isochrone, request);
   loki_worker.isochrones(request);
 
   // Process isochrone request
-  auto result = thor_worker.isochrones(request);
+  auto response_json = thor_worker.isochrones(request);
 
-  // Check if the result contains the expected string
-  EXPECT_NE(result.find(expected), std::string::npos) << "isochrones failed: expected " + expected;
+  rapidjson::Document response;
+  response.Parse(response_json);
```

Comment:
Bit confusing that `response_json` is a string but `response` is a json object. I'll leave it to you if you want to change. Rest of it LGTM! 👍